spec/game_list_spec.rb

```diff
@@ -1,38 +1,46 @@
 require 'spec_helper'
 
 RSpec.describe GameList do
-  it "can create a new Game List class instance" do
-    game_list = GameList.new('./data/games_subset.csv', 'stat_tracker')
+  before(:each) do
+    locations = {
```

Comment:
This is a minor tweak, but for these specs I don't think we have to include all 3 keys for the locations hash - so in this case I think we only need games and we could reduce this to just:
```
locations = {
      games: "./data/games_subset.csv",
    }
```

Reply:
I second this. Only add what you need :)

Reply:
We want to have all three because some of the later methods need access to all three csv files. Currently the methods dont need them but they would if we got that far.